.agents/skills/appwrite-typescript/SKILL.md

```
@@ -0,0 +1,706 @@
+    });
+
+    res.cookie('a_session_[PROJECT_ID]', session.secret, {
+        httpOnly: true, secure: true, sameSite: 'strict',
+        expires: new Date(session.expire), path: '/',
+    });
+    res.json({ success: true });
+});
+```
+
+> **Cookie security:** Always use `httpOnly`, `secure`, and `sameSite: 'strict'` to prevent XSS. The cookie name must be `a_session_<PROJECT_ID>`.
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)